src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -171,6 +171,23 @@ public void Should_blow_up_if_name_is_null()
             Assert.Throws<ArgumentNullException>(() => creator.CreateQueueIfNecessary(bindings, WindowsIdentity.GetCurrent().Name));
         }
 
+        [Test]
+        public void Verify_msmq_permissions_and_delete_has_lower_lenght_limit_compared_to_queue_name()
+        {
+            //increase to 50 and create will work but SetPermission and Delete will blow up
```

Comment:
@andreasohlund So this test is documenting a limit? If so, shouldn't we then also test failure on lenght = 50?

Also, as the queue name length is allowed this seems a limit on the SetPermissions method so a limit in the .net framework. Did we ever report a bug report for it? We should do that, link it here with the test testing on expected failure?